x-pack/filebeat/input/streaming/websocket.go

```diff
@@ -39,13 +39,20 @@ type websocketStream struct {
 	tokenSource oauth2.TokenSource
 	tokenExpiry <-chan time.Time
 	time        func() time.Time
+	keepAlive   *keepAlive
 }
 
 type loggingRoundTripper struct {
 	rt  http.RoundTripper
 	log *logp.Logger
 }
 
+type keepAlive struct {
+	cfg     config
```

Comment:
```suggestion
	cfg     KeepAlive
```
with associated `s/.KeepAlive//g`.